dist/vespa.spec

```diff
@@ -586,6 +586,10 @@ exit 0
 %systemd_postun_with_restart vespa-configserver.service
 %endif
 
+%post base
+
+ln -sf %{_prefix}/var/tmp %{_prefix}/tmp
```

Comment:
This might work as expected if `%{_prefix}/tmp` is a directory, since it would make a symlink from `%{_prefix}/tmp/tmp` to `%{_prefix}/var/tmp`